runtimes/js/encore.dev/api/mod.ts

```diff
@@ -19,6 +19,71 @@ export type Method =
   | "TRACE"
   | "CONNECT";
 
+export type HttpStatus =
```

Comment:
Can we move this to the bottom of the file? :)

Reply:
yeah sure! I was contemplating moving it to a separate file even

Reply:
I moved it to a separate file so that where I defined all codes with their names, so this will work:

```ts
export const get = api(
  { expose: true, method: "GET", path: "/hello/:name" },
  async (req: Request): Promise<Response> => {
    // ...
    return { message: msg, status: HttpStatus.Created };
  },
);
interface Response {
  message: string;
  status: HttpStatus;
}

```

wdyt?